cocotb/share/makefiles/Makefile.inc

```diff
@@ -92,11 +92,13 @@ else ifeq ($(OS),Linux)
 GXX_ARGS	+= -Werror -Wcast-qual -Wcast-align -Wwrite-strings \
 			-Wall -Wno-unused-parameter \
 			-fno-common -g -DDEBUG -fpic
-GCC_ARGS	= $(GXX_ARGS) -Wstrict-prototypes -Waggregate-return
+GCC_ARGS	:= $(GXX_ARGS) -Wstrict-prototypes -Waggregate-return
```

Comment:
This is by itself a bug - note all the other cases use `:=` to perform the copy. If desirable, I can move this to its own PR.

Reply:
Done in #1162